### ISSUE TITLE
Sharing the kernel between a console and notebook sessions

### ISSUE THREAD

**Issue by tfabbri** [jupyterlab/jupyterlab#4538]
Dear Jupyterlab team, 

is that possible/planned to have a console session 'attached'  to the same kernel of a notebook within jupyterlab? 

From my perspective this could help the user (for sure me :) ) in perform a more effective debugging/development of the code, without requiring the creation of several cells (which will be removed, because used just to test). In this way,  the notebook will be more focused on providing the real results.

This request maybe is related to the issue #2044

**Comment by tfabbri** [comment]
Yes, this does suffice. I did not know this feature was already implemented. Thank you so much!  

But by right clicking on the notebook I got the following options

![untitled](https://user-images.githubusercontent.com/3789256/39801070-3cf15020-536a-11e8-9488-5501aca55ec9.png)

and the "New Console for Notebook" is not available. 

I had access to this functionality through Commands tab => New Console and there I can select the kernel to use. Otherwise, I did not find other ways to this.
I am using Jupyterlab 0.32.1 ship with Anaconda Cloud.
